test/wdio/svg-class/cmp.test.tsx

```diff
@@ -18,10 +31,7 @@ describe('svg class', () => {
     expect(circle.getAttribute('class')).toBe(null);
     expect(rect.getAttribute('class')).toBe(null);
 
-    const button = app.querySelector('button');
-    button.click();
-
-    await waitForChanges();
+    await $('button').click();
```

Comment:
FWIW, I had some flakiness trying to run this test locally - adding the following helped:
```suggestion
    await $('button').click();
    await $('svg-class').waitForStable();
```